src/test/java/org/kcctl/command/GetPluginsCommandTest.java

```diff
@@ -37,33 +40,61 @@
 @DisplayNameGeneration(ReplaceUnderscores.class)
 class GetPluginsCommandTest extends IntegrationTest {
 
+    private static final Version SPANNER_CONNECTOR_MIN_VERSION = new Version(2, 1);
+    private static final Version MIRROR_MAKER_MIN_GRANULAR_VERSION = new Version(3, 2);
+    private static final Version FILE_CONNECTOR_REMOVAL_MIN_VERSION = new Version(3, 2); // Also removed in 3.1.1 (see https://issues.apache.org/jira/browse/KAFKA-13748) but there's no Debezium image built off of that so 3.2.x is fine
+
     @InjectCommandContext
     KcctlCommandContext<GetPluginsCommand> context;
 
     @Test
     public void should_list_source_plugins() throws Exception {
-        String debeziumVersion = ContainerImageVersions.getStableVersion("debezium/connect");
+        String debeziumVersion = getDebeziumVersion();
         String kafkaVersion = getConnectVersion();
 
+        Version parsedDebeziumVersion = new Version(debeziumVersion);
+        Version parsedKafkaVersion = new Version(kafkaVersion);
+
+        String spannerConnectorDescription = null;
+        if (parsedDebeziumVersion.greaterOrEquals(SPANNER_CONNECTOR_MIN_VERSION)) {
+            spannerConnectorDescription = "source   io.debezium.connector.spanner.SpannerConnector              " + debeziumVersion;
+        }
```

Comment:
+1 for switching to `connect-base`, although we'd still end up doing most of the dancing here since the rest (besides the Spanner connector) concerns MM2 and the file connectors :)

Reply:
LOL, yeah, fair point. The other advantage would be faster test execution, as the plug-in path scanning in Connect feels surprisingly slow.

Reply:
Noice!